src/popup/router/components/SwapRoute.vue

```diff
@@ -92,6 +92,7 @@ export default {
     .divider {
```

Comment:
`SwapRoute` component shouldn't be in scope of this PR.

Reply:
I've disabled the tooltip and added a loader

Reply:
Should be in a separate commit.